tests/test_modeling_tf_common.py

```diff
@@ -1296,6 +1296,56 @@ def test_loss_computation(self):
 
                 self.assertEqual(loss.shape, [loss_size])
 
+    def test_keras_fit(self):
+        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+        for model_class in self.all_model_classes:
+            model = model_class(config)
+            if getattr(model, "hf_compute_loss", None):
```

Comment:
Out of curiosity -- are there models w/o `hf_compute_loss`?

Reply:
I don't believe so - this code might be vestigial. I'll check tomorrow!